### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by DrRoetker** [Tobi823/ffupdater#108]
Hi guys,
I guess FFUpdater was running in the background, as I received a crash report with the request to post it here. Hope this helps!

Stacktrace:
```
de.marmaro.krt.ffupdater.app.impl.exceptions.NetworkException: Fail to consume de.marmaro.krt.ffupdater.app.impl.fetch.github.GithubConsumer.Release from 'https://api.github.com/repos/bromite/bromite/releases/latest'.
	at de.marmaro.krt.ffupdater.app.impl.fetch.ApiConsumer.readNetworkResource(ApiConsumer.kt:61)
	at de.marmaro.krt.ffupdater.app.impl.fetch.ApiConsumer.access$readNetworkResource(ApiConsumer.kt:23)
	at de.marmaro.krt.ffupdater.app.impl.fetch.ApiConsumer$consumeNetworkResource$2.invokeSuspend(ApiConsumer.kt:38)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
Caused by: java.net.UnknownHostException: Unable to resolve host "api.github.com": No address associated with hostname
	at java.net.Inet6AddressImpl.lookupHostByName(Inet6AddressImpl.java:156)
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:103)
	at java.net.InetAddress.getAllByName(InetAddress.java:1152)
	at com.android.okhttp.Dns$1.lookup(Dns.java:41)
	at com.android.okhttp.internal.http.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:178)
	at com.android.okhttp.internal.http.RouteSelector.nextProxy(RouteSelector.java:144)
	at com.android.okhttp.internal.http.RouteSelector.next(RouteSelector.java:86)
	at com.android.okhttp.internal.http.StreamAllocation.findConnection(StreamAllocation.java:176)
	at com.android.okhttp.internal.http.StreamAllocation.findHealthyConnection(StreamAllocation.java:128)
	at com.android.okhttp.internal.http.StreamAllocation.newStream(StreamAllocation.java:97)
	at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:289)
	at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:232)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:465)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:411)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getHeaders(HttpURLConnectionImpl.java:167)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getHeaderField(HttpURLConnectionImpl.java:211)
	at java.net.URLConnection.getContentEncoding(URLConnection.java:551)
	at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getContentEncoding(DelegatingHttpsURLConnection.java:143)
	at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getContentEncoding(HttpsURLConnectionImpl.java:30)
	at de.marmaro.krt.ffupdater.app.impl.fetch.ApiConsumer.readNetworkResource(ApiConsumer.kt:49)
	... 8 more
Caused by: android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associated with hostname)
	at libcore.io.Linux.android_getaddrinfo(Native Method)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:136)
	at libcore.io.BlockGuardOs.android_getaddrinfo(BlockGuardOs.java:221)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:136)
	at java.net.Inet6AddressImpl.lookupHostByName(Inet6AddressImpl.java:135)
	... 27 more
```
Device information:
| key | value |
| --- | --- |
| version name | 75.0.0 |
| version code | 91 |
| build type | release |
| SDK version | 31 |
| android release | 12 |
| board | redfin |
| brand | google |
| device | redfin |
| manufacturer | Google |
| model | Pixel 5 |
| product | redfin |
| SOC manufacturer | Qualcomm |
| SOC model | SM7250 |
| supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| tags | release-keys |
| build time | 1642293234000 |

**Comment by Tobi823** [comment]
I will take a closer look at the problem after the university exams. 

But it's nothing critical. your smartphone tried to check for new updates when it had no internet connection.
Unfortunately, it is very difficult to avoid such errors on Android (across every Android version and every Android device).

**Comment by wildwestrom** [comment]
Same error here.

**Comment by Tobi823** [comment]
Will be fixed in 75.2.0
